src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -102,15 +107,21 @@ const UserProfile = ({
     );
   };
   return (
-    <View style={styles.container}>
-      {userProfile()}
-      <ToggleSwitchAnon
-        value={isAnonymous}
-        onValueChange={() => setTypeUser(!isAnonymous)}
-        labelLeft="Anonymity"
-        labelColor={COLORS.gray8}
-      />
-    </View>
+    <>
+      <View style={styles.container}>
+        {userProfile()}
+        <ToggleSwitch
+          value={isAnonymous}
+          onValueChange={() => setTypeUser(!isAnonymous)}
+          labelLeft="Anonymity"
+          backgroundActive={colors.lightgrey}
+          backgroundInactive={colors.lightgrey}
+          circleInActiveColor={colors.blue1}
+          inactiveTextColor={colors.blue1}
+          styleLabelLeft={styles.switch}
+        />
+      </View>
+    </>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ToggleSwitch` component has been updated with new props, replacing the old `ToggleSwitchAnon` component. The new props include `backgroundActive`, `backgroundInactive`, `circleInActiveColor`, `inactiveTextColor`, and `styleLabelLeft`, which were not present in the old component. This change is consistent with the PR description and there are no issues with these changes.
```diff
-      <ToggleSwitchAnon
-        value={isAnonymous}
-        onValueChange={() => setTypeUser(!isAnonymous)}
-        labelLeft="Anonymity"
-        labelColor={COLORS.gray8}
-      />
+      <ToggleSwitch
+        value={isAnonymous}
+        onValueChange={() => setTypeUser(!isAnonymous)}
+        labelLeft="Anonymity"
+        backgroundActive={colors.lightgrey}
+        backgroundInactive={colors.lightgrey}
+        circleInActiveColor={colors.blue1}
+        inactiveTextColor={colors.blue1}
+        styleLabelLeft={styles.switch}
+      />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->